### PR TITLE
always display awx_status_total

### LINKED ISSUE #12965: Make awx_status_total items always report a value (0 if none exist in a status)
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Feature type

Enhancement to Existing Feature

### Feature Summary

Right now, if there are no jobs in a system or no jobs currently in a status, then the values of `awx_status_total` change.

For example, if there are no metrics, `awx_status_total` is not shown on `api/v2/metrics`

If there are no canceled jobs, then we don't see that value:
```
awx_status_total{status="pending"} 5.0
awx_status_total{status="successful"} 1.0
awx_status_total{status="running"} 55.0
awx_status_total{status="error"} 1.0
```

Similarly, if we don't have any pending jobs, that value disappears.

This causes weirdness from a monitoring perspective because it can make dashboards and alerts have problems if the keys do not exist in the data, or if there are areas where we are missing values.

Instead of this, always return values, even if they are zero:

```
awx_status_total{status="pending"} 5.0
awx_status_total{status="successful"} 1.0
awx_status_total{status="running"} 55.0
awx_status_total{status="error"} 1.0
awx_status_total{status="canceled"} 0.0
```

### Select the relevant components

- [ ] UI
- [X] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [ ] Other

### Steps to reproduce

GET to /api/v2/metrics

### Current results

Sometimes items missing

### Sugested feature result

All statuses present all the time

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kdelee**
##### SUMMARY

Always report some number (even 0) for each job status. This way we don't have null data in monitoring data this makes writing alerts and dashboards easier
fixes https://github.com/ansible/awx/issues/12965

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - API

### CODE REVIEW

File: awx/main/analytics/metrics.py [hunk 1]
```diff
@@ -169,8 +169,9 @@ def metrics():
 
     all_job_data = job_counts(None)
     statuses = all_job_data.get('status', {})
-    for status, value in statuses.items():
-        STATUS.labels(status=status).set(value)
+    states = ('pending', 'waiting', 'running', 'successful', 'failed', 'error', 'canceled')
```
Comment (AlanCoding) [inline awx/main/analytics/metrics.py]: `set(dict(UnifiedJob.STATUS_CHOICES).keys()) - set(['new'])`?

Comment (kdelee) [reply awx/main/analytics/metrics.py]: sounds good will do

**Comment by kdelee** [comment]
currently passing tests and this is output:
```
# HELP awx_status_total Status of Job launched
# TYPE awx_status_total gauge
awx_status_total{status="waiting"} 0.0
awx_status_total{status="successful"} 0.0
awx_status_total{status="failed"} 0.0
awx_status_total{status="error"} 0.0
awx_status_total{status="pending"} 0.0
awx_status_total{status="canceled"} 0.0
awx_status_total{status="running"} 0.0
```

**Review by AlanCoding — APPROVED** [review]

**Review by rebeccahhh — APPROVED** [review]

**Comment by kdelee** [comment]
image

**Pull request merged**